silasdk/message.py

```diff
@@ -109,7 +109,7 @@ def postRequest(app: App, path: str, msg_type: str, payload: dict, key: Optional
         key :user_private_key
     """
     data = createMessage(app, payload, msg_type)
-    print(data)
+    logger.debug(data)
```

Comment:
No access to `self` here, so no way to determine desired logging level.  But I'm in agreement here that suppressing this message by default is appropriate behavior, and the SDK consumer can set the default level to DEBUG as desired.